environments/openstack/playbook-bootstrap-basic.yml

```diff
@@ -401,3 +426,12 @@
         state: present
         server: test
         volume: test
+
+    - name: Create load balancer
+      openstack.cloud.loadbalancer:
+        cloud: test
+        state: present
+        name: test
+        vip_network: test
+        auto_public_ip: true
+        public_network: public
```

Comment:
If you want to test something, you also need to add a listener, pool and member, like forward SSH to the instance we are creating? I'm not sure if even an amphora will be spawned by the current state.

Reply:
Amphora is spawned, I had just tested. But it does nothing.